addons/hr/models/res_company.py

```diff
@@ -9,3 +9,4 @@ class Company(models.Model):
 
     hr_presence_control_email_amount = fields.Integer(string="# emails to send")
     hr_presence_control_ip_list = fields.Char(string="Valid IP addresses")
+    employee_properties_definition = fields.PropertiesDefinition('Employee Properties')
```

Comment:
Do we really want to add more things onto the company?

Also, the commit message mentions that it depends on the "hr.company" model, but it's "res.company".